cwltool/expression.py

```diff
@@ -126,7 +127,7 @@ def next_seg(remain, obj):  # type: (Text, Any) -> Any
             try:
                 key = int(m.group(0)[1:-1])
             except ValueError as v:
-                raise WorkflowException(unicode(v))
+                raise WorkflowException(u(v))
```

Comment:
`make mypy` and https://travis-ci.org/common-workflow-language/cwltool/jobs/199538581#L185